### PR TITLE
38 edit main class v1.0

### LINKED ISSUE #38: Edit main class to run application
Rename main java class and print greeting and goodbye message

### PR DESCRIPTION
**Pull request by DextheChik3n**
To enable starting the CafeCtrl application and terminate with the "bye" command

Fixes #38

### CODE REVIEW

File: src/main/java/seedu/cafectrl/CafeCtrl.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import seedu.cafectrl.parser.Parser;
+import seedu.cafectrl.ui.Ui;
+
+public class CafeCtrl {
+    private final Ui ui;
+    private final Menu menu;
+    private Command command;
+
+    /**
+     * Private constructor for the CafeCtrl class, used for initializing the user interface and menu list.
```
Comment (Cazh1) [inline src/main/java/seedu/cafectrl/CafeCtrl.java:15]: Perhaps you meant public constructor?

File: src/main/java/seedu/cafectrl/command/ExitCommand.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package seedu.cafectrl.command;
+
+import seedu.cafectrl.data.Menu;
+import seedu.cafectrl.ui.Ui;
+
+public class ExitCommand extends Command{
```
Comment (Cazh1) [inline src/main/java/seedu/cafectrl/command/ExitCommand.java:6]: Noticed missing spacing before curly bracket {

File: src/main/java/seedu/cafectrl/ui/Ui.java [hunk 3]
```diff
@@ -16,7 +16,12 @@ public Ui() {
         scanner = new Scanner(System.in);
     }
 
+    public void printLine() {
+        showToUser("-----------------------------------------------------");
```
Comment (Cazh1) [inline src/main/java/seedu/cafectrl/ui/Ui.java:20]: Perhaps you can consider using a final static String instead of "magic String"?

**Review by Cazh1 — COMMENTED** [review]
Suggested some changes

File: src/main/java/seedu/cafectrl/CafeCtrl.java (hunk 1 above)
Comment (DextheChik3n) [reply src/main/java/seedu/cafectrl/CafeCtrl.java:15]: Sorry I was suppose to declare the CafeCtrl constructor as private... I'm going to change that now thank you!

File: src/main/java/seedu/cafectrl/command/ExitCommand.java (hunk 2 above)
Comment (DextheChik3n) [reply src/main/java/seedu/cafectrl/command/ExitCommand.java:6]: oops thank you for catching that

File: src/main/java/seedu/cafectrl/CafeCtrl.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            try {
+                String fullUserInput = ui.receiveUserInput();
+                command = Parser.parseCommand(menu, fullUserInput);
+                command.execute(menu, ui);
+            } catch (Exception e) {
+                ui.showToUser(e.getMessage());
+            } finally {
+                ui.printLine();
+            }
+        } while (!(command instanceof ExitCommand)); //end the program if the command obj is a ExitCommand
```
Comment (ziyi105) [inline src/main/java/seedu/cafectrl/CafeCtrl.java:48]: Perhaps you should avoid using instanceof?

File: src/main/java/seedu/cafectrl/command/ExitCommand.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    public static final String COMMAND_WORD = "bye";
+    @Override
```
Comment (ziyi105) [inline src/main/java/seedu/cafectrl/command/ExitCommand.java:8]: Maybe you can have one empty line here to make a distinction between attributes and method

**Review by ziyi105 — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/cafectrl/CafeCtrl.java (hunk 4 above)
Comment (DextheChik3n) [reply src/main/java/seedu/cafectrl/CafeCtrl.java]: Ok i changed it to include a isExit() method for Command and ExitCommand class

**Review by ziyi105 — APPROVED** [review]
lgtm

**Pull request merged**